CODEOWNERS

```diff
@@ -1 +1 @@
-* @volovyk-s @ailisp
+* @marcus-pousette
```

Comment:
You should be a part of NEAR Inc to become a maintainer ;) Ping me if you are interested.

Reply:
Hey thanks for the response! I see. As I needed the changes quite urgently I started to maintain my own fork [https://github.com/the-solvei/borsh-ts](https://github.com/the-solvei/borsh-ts). I fixed a few bugs and also made additional improvements (which means this PR is outdated). I would have to rebase those changes to this branch. If you would want to play around with a working solution I would recommend you to try out my forked version. 

I will give you a ping, regarding maintainer role 👍

Reply:
Awesome! I will take a look.
Ping me after you rebase it.

Reply:
Hey @marcus-pousette, any plans to publish to npm?

Reply:
@AlaaZorkane. Sorry for late reply. Yes! [it is live](https://github.com/dao-xyz/borsh-ts)